docs/README.md

```diff
@@ -43,6 +43,10 @@
     - [Handle requests](./network.md#handle-requests)
     - [Modify requests](./network.md#modify-requests)
     - [Abort requests](./network.md#abort-requests)
+1. [Assertions](./assertions.md)
```

Comment:
Assertions and Verification - kind of the same thing?

Reply:
Yes, there's an overlap, but I've seen users ask for assertions even after having seen our docs. Planning to remove the "assertion" related bits from the verification doc (like `page.$eval`). Verification will focus on screenshots, bubbling console logs, network failures, errors.